### PR TITLE
Use a release version of the routes controller (v0.0.1)

### PR DESCRIPTION
**Pull request by guillaumerose**
https://github.com/code-ready/routes-controller/releases/tag/v0.0.1

### CODE REVIEW

**Comment by guillaumerose** [comment]
/hold
I forgot that I need to use a tag like :current to avoid the duplication of version in snc and crc.

**Review by gbraad — APPROVED** [review]

**Comment by guillaumerose** [comment]
I tried with `:current` tag. It works but I am worry that the user might delete the image from the bundle by mistake. 
If the user does something like `podman rmi $(podman images -q)`, the pod won't be able to start anymore as the tag is only local.

/unhold

**Comment by gbraad** [comment]
That would be a user error.m the vm is supposed to be tested ephemeral, so a delete and restart would resolve that

**Comment by praveenkumar** [comment]
I am thinking it https://github.com/code-ready/snc/pull/398#issuecomment-827550149 this way.

File: pkg/crc/machine/start.go [hunk 1]
```diff
@@ -661,7 +661,7 @@ func ensureRoutesControllerIsRunning(sshRunner *crcssh.Runner, ocConfig oc.Confi
 			Containers: []v1.Container{
 				{
 					Name:            "routes-controller",
-					Image:           "quay.io/crcont/routes-controller:6effc2f0304d3a11c1e22c278b10630e834d0220",
+					Image:           "quay.io/crcont/routes-controller:latest",
```
Comment (anjannath) [inline pkg/crc/machine/start.go:664]: 👍🏽

**Review by anjannath — APPROVED** [review]

**Comment by anjannath** [comment]
retest this please

**Pull request merged**